docs/samples/Microsoft.ML.AutoML.Samples/Sweepable/SearchSpaceExample.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Text;
+using System.Text.Json;
+using Microsoft.ML.SearchSpace;
+using Microsoft.ML.SearchSpace.Option;
+
+namespace Microsoft.ML.AutoML.Samples
+{
+    public static class SearchSpaceExample
+    {
+        public static void Run()
+        {
+            // The following code shows how to create a SearchSpace for MyParameter.
+            var myParameterSearchSpace = new SearchSpace<MyParameter>();
+
+            // Equivalently, you can also create myParameterSearchSpace from scratch.
+            var myParameterSearchSpace2 = new SearchSpace.SearchSpace();
+
+            // numeric options
+            myParameterSearchSpace2["IntOption"] = new UniformIntOption(-10, 10, false, 0);
+            myParameterSearchSpace2["SingleOption"] = new UniformSingleOption(1, 10, true, 1);
+            myParameterSearchSpace2["DoubleOption"] = new UniformDoubleOption(-10, 10, false, 0);
```

Comment:
It might help to add the param names for these to make it easier to see what each of the numbers and values represent.